docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -39,6 +39,8 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | - | - | maxSizeOfNetworkAddressAlias|Max size of network address detected in the be monitored system.| - | 1_000_000|
 | - | - | maxPageSizeOfQueryProfileSnapshot|The max size in every OAP query for snapshot analysis| - | 500 |
 | - | - | maxSizeOfAnalyzeProfileSnapshot|The max number of snapshots analyzed by OAP| - | 12000 |
+| - | - | syncThreads|The number of threads used to synchronously refresh the metrics data to the storage. When the value is zero or negative, the number of processors will used as the default value.| SW_CORE_SYNC_THREADS | -1 |
```

Comment:
Your default value is -1. So your explanation doesn't make sense. What exactly threads are you going to use in default?